src/customError/errorMap.js

```diff
@@ -0,0 +1,22 @@
+const EP1000 = 'EP1000'
+const EP1001 = 'EP1001'
+const EA1000 = 'EA1000'
+const EA1001 = 'EA1001'
+const EA1002 = 'EA1002'
+const EA1003 = 'EA1003'
+const EA1004 = 'EA1004'
+const EA1005 = 'EA1005'
+const EA1006 = 'EA1006'
+const EA1007 = 'EA1007'
+
+module.exports = new Map([
+    [EP1000, 'the command did not have the correct seperators. Please try again'],
+    [EP1001, 'the command did not have the correct seperators. Please try again'],
+    [EA1000, 'the username you specified does not exist for the team'],
+    [EA1001, 'the project specified does not exist'],
```

Comment:
these read like an explanation, you should also tell users what action they should take next. For example,
"the project specified does not exist. Make sure you've spelled the project name correctly"